KerbalStuff/objects.py

```diff
@@ -74,8 +76,21 @@ def check_password(self, password: str) -> bool:
     def create_confirmation(self) -> None:
         self.confirmation = binascii.b2a_hex(os.urandom(20)).decode('utf-8')
 
-    def __repr__(self) -> str:
-        return '<User %r>' % self.username
+    def base_path(self) -> str:
+        return secure_filename(self.username) + '_' + str(self.id)
+
+    def background_url(self, protocol: Optional[str], cdn_domain: Optional[str]) -> Optional[str]:
+        if not self.backgroundMedia:
+            return None
+        # Fix old db data
+        # str.removeprefix is only available in Python3.9+
+        if self.backgroundMedia.startswith('/content/'):
+            self.backgroundMedia = self.backgroundMedia[8:]
```

Comment:
In 4 places we have `[9:]`, in 2 places we have `[8:]`. Is the inconsistency intentional?

Reply:
Should be the same, of course. I could've sworn I changed those two to 9 at some point...